ExeterTeam25/group-software-project
# Deleting posts

What I did: 
- Let users delete their own posts. 

To test: 
- Make sure you have a post. 
- Delete that post. 
- That post should be gone!
